include/deal.II/matrix_free/fe_point_evaluation.h

```diff
@@ -1349,6 +1349,22 @@ class FEPointEvaluation
   Tensor<1, spacedim, Number>
   normal_vector(const unsigned int point_index) const;
 
+  /**
+   * Return the normal gradient in real coordinates at the point with index
+   * `point_index` after a call to FEPointEvaluation::evaluate() with
+   * EvaluationFlags::gradients set.
+   */
+  const value_type
+  get_normal_derivative(const unsigned int point_index) const;
+
+  /**
+   * Write a contribution that is tested by the normal gradient to the field
```

Comment:
Do we still write `normal gradients`? I would prefer to use the term `normal derivative`, unless you have concerns that this could be confused somehow.

Reply:
I would also like to introduce a function for normal Hessians (or normal second derivatives) for `FEFaceEvaluation`. I personally prefer the name normal gradients but for the library, it is important to stay consistent in naming. I can change it here if you want.